lib/smart_city/event/_metadata.ex

```diff
@@ -0,0 +1,60 @@
+defmodule SmartCity.Event.DatasetUpdate.Metadata do
+  @moduledoc """
+  A struct defining internal metadata on a registry event message.
```

Comment:
Update comment

Reply:
to avoid confusion between the file/directory name and the module name, should these be moved into a `lib/smart_city/event/dataset_update/` directory?

Reply:
your pr feedback is our command